_posts/2018-09-24-galaxy-hts-data-analysis-workshop.md

```diff
@@ -0,0 +1,22 @@
+---
+site: freiburg
+title: A successuful Galaxy HTS data analysis workshop
+location: Freiburg, Germany
+tags: [training]
+---
+
+<div class="multiple-img">
+<a data-flickr-embed="true" data-header="true" data-footer="true"  href="https://www.flickr.com/photos/134305289@N03/43888095455/in/datetaken/" title="Galaxy HTS data analysis workshop @ Freiburg"><img src="https://farm2.staticflickr.com/1892/43888095455_3238320d36_z.jpg" width="640" height="427" alt="Galaxy HTS data analysis workshop @ Freiburg"></a><script async src="//embedr.flickr.com/assets/client-code.js" charset="utf-8"></script>
+</div>
+
+A new Galaxy workshop on HTS data analysis was held in Freiburg last week. During this week, the [Freiburg Galaxy Team](https://usegalaxy.eu/freiburg/) and [Bionformatics facility from the MPI-IE Freiburg](https://www.ie-freiburg.mpg.de/bioinformaticsfac) offered an introductory course to the analysis of high-throughput sequencing data using Galaxy:
```

Comment:
Could you move this one above the images? Then it will show up in the "preview" of the news list like https://usegalaxy-eu.github.io/news